packages/shared/src/hooks/useContextMenu.ts

```diff
@@ -2,7 +2,7 @@ import { MouseEventHandler, useCallback, useMemo } from 'react';
 import { useContextMenu as useContexifyContextMenu } from '@dailydotdev/react-contexify';
 import { useQuery, useQueryClient } from '@tanstack/react-query';
 import { generateQueryKey, RequestKey } from '../lib/query';
-import { useViewSize, ViewSize } from './useViewSize';
+import { useViewSize, ViewSize } from '.';
```

Comment:
Needed to change the import path, so that we can mock `useViewSize` in tests properly. Otherwise it detects a mobile screen and existing tests fail.